README.md

```diff
@@ -1,4 +1,37 @@
-# uCredit-API
+## uCredit-API
+This is the backend for [uCredit](https://ucredit.me), 4-year course planning, streamlined. Deployed on [Render](render.com).
+
+- Development deployment: https://ucredit-dev.onrender.com/
+- Production deployment: https://ucredit-api.onrender.com/
+
+Please allow a few seconds for the server to start up. 
+
+## Structure
+- API routes saved in ./routes
+- Mongoose Schema saved in ./model
+- Tests saved in ./tests
+- Database connection made in ./data/db.js
+
+### Run locally
+1. Clone this repository. 
+```
+git clone https://github.com/uCredit-Dev/uCredit-API.git
+``` 
+2. Install dependencies. 
+```
+npm i
+``` 
+3. Create .env file at the root. Copy paste .env for development! (Ask a dev for access)
+```
+touch .env
+``` 
+4. Start server.
+```
+npm start
+``` 
+5. Navigate to http://localhost:4567/
+
+### Run locally with docker 
 
 To run program from a fresh clone:
 
```

Comment:
For step 1, do we need to create the .env file again? Maybe modify it to "Update `.env` file with the following:" or "Make sure the `.env` file is created and filled (from earlier step)."

Reply:
Otherwise looks amazing!